addons/mail/static/src/components/chat_window_header/chat_window_header.xml

```diff
@@ -2,78 +2,78 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.ChatWindowHeader" owl="1">
-        <div class="o_ChatWindowHeader" t-att-class="{ 'o-mobile': messaging and messaging.device.isMobile }" t-attf-class="{{ className }}" t-on-click="_onClick" t-ref="root">
+        <div class="o_ChatWindowHeader d-flex align-items-center text-white" t-att-class="{ 'o-mobile': messaging and messaging.device.isMobile }" t-attf-class="{{ className }}" t-on-click="_onClick" t-ref="root">
             <t t-if="chatWindow">
                 <t t-if="props.hasCloseAsBackButton">
-                    <div class="o_ChatWindowHeader_command o_ChatWindowHeader_commandBack o_ChatWindowHeader_commandClose" t-att-class="{ 'o-mobile': messaging.device.isMobile }" t-on-click="_onClickClose" title="Close conversation">
+                    <div class="o_ChatWindowHeader_command o_ChatWindowHeader_commandBack o_ChatWindowHeader_commandClose d-flex align-items-center h-100 px-3 py-0 mr-2" t-att-class="{ 'o-mobile': messaging.device.isMobile }" t-on-click="_onClickClose" title="Close conversation">
                         <i class="fa fa-arrow-left"/>
                     </div>
                 </t>
                 <t t-if="chatWindow.thread and chatWindow.thread.model === 'mail.channel'">
                     <ThreadIcon
-                        className="'o_ChatWindowHeader_icon o_ChatWindowHeader_item'"
+                        className="'o_ChatWindowHeader_icon o_ChatWindowHeader_item mx-1 my-0'"
```

Comment:
Unfortunately `mx-1` will invalidate the `:first-child` raw SCSS rule (https://github.com/odoo/odoo/pull/85842/files#diff-1070df8795bb28ca9c7104534ecd9e28ae846dd19b248cbe69b4f6ae9e5ee481L38).

The same for the following element in case `ThreadIcon` is not rendered.
I'd suggest tu use padding on the container instead... or conditionally switch between `mx-1` and `ms-3 me-1` depending by the circumstances... to be tested

We aware of RTL ;)